src/openrct2/common.h

```diff
@@ -71,6 +71,10 @@ using colour_t = uint8_t;
 #        define RCT2_ENDIANESS __ORDER_LITTLE_ENDIAN__
 #        define LOBYTE(w) ((uint8_t)(w))
 #        define HIBYTE(w) ((uint8_t)(((uint16_t)(w) >> 8) & 0xFF))
+#    else
+#        define RCT2_ENDIANESS __ORDER_BIG_ENDIAN__
+#        define HIBYTE(w) ((uint8_t)(w))
+#        define LOBYTE(w) ((uint8_t)(((uint16_t)(w) >> 8) & 0xFF))
```

Comment:
@IntelOrca these are required, I added `#error` in case `RCT2_ENDIANESS` is not defined and `HIBYTE` and `LOBYTE` are used somewhere.